bedrock/firefox/tests/test_helpers.py

```diff
@@ -57,10 +58,12 @@ def test_button_force_direct(self):
         directly to https://download.mozilla.org.
         """
         rf = RequestFactory()
+        locale = 'fr'
         get_request = rf.get('/fake')
-        get_request.locale = 'fr'
+        get_request.locale = locale
```

Comment:
This is a lot of code change. Had you not already done this I would have suggested making yourself a helper function like:

```python
def get_l10n(locale):
    return fluent_l10n([locale, 'en'], settings.FLUENT_DEFAULT_FILES)
```

And then just replacing the line below with `'fluent_l10n': get_l10n(get_request.locale)`. But since you've already done the work I don't think you should refactor now. It looks good.

Reply:
I'm happy to make the change, thanks for the advice

Reply:
totally up to you if you just want the diff to be smaller. what you've done here is fine with me though.